### DISCUSSION TITLE
Développeur SPHINX

### DISCUSSION THREAD

**Discussion by SFasseu** [community/community#144761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

**Titre du poste :** Développeur SPHINX

**Description du poste :**
Nous recherchons un développeur expérimenté dans l’utilisation de SPHINX pour Réfonte site internet. Le rôle inclut la configuration, la personnalisation et l’optimisation.

**Compétences requises :**
- Maîtrise de SPHINX.
- Créer des questionnaires structurés et bien conçus.
- Utiliser les fonctionnalités avancées de Sphinx pour le traitement de données.
- Analyser et interpréter de manière critique les données recueillies.
- Présenter les résultats de manière cohérente et académique.
- Maîtriser les bonnes pratiques en matière de collecte et de gestion de données avec Sphinx.
- Minimum 3ans expériences.

**Lieu :** Télétravail
**Durée :** Freelance

**Comment postuler :**
Veuillez soumettre votre candidature en répondant à cette issue ou en nous contactant (Personal info removed by Staff)

Merci de votre intérêt !

**Comment by Akash1134** [comment]
Hi @SFasseu , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
